parm/config/config.base.emc.dyn

```diff
@@ -136,17 +140,18 @@ export SENDDBN_NTC=${SENDDBN_NTC:-"NO"}
 export SENDDBN=${SENDDBN:-"NO"}
 export DBNROOT=${DBNROOT:-${UTILROOT}/fakedbn}
 
+export print_esmf=".false."
+export esmf_profile=".true."
```

Comment:
aren't these used only in the forecast components?
If so, why not in `config.fcst`?

Reply:
@WalterKolczynski-NOAA @JessicaMeixner-NOAA please confirm if these are only needed in the forecast job(s), if so I will move these two variables to config.fcst. Thanks!

Reply:
Only needed for forecast, and esmf_profile should probably default to false (not saying it'snot true in f/c-c but probably just an oversight).

Reply:
Yes, these only matter when the model is run (so I guess for EnKF as well). The first controls whether the individual PET log files are written into the run directory. The second controls whether the models are profiled to identify bottlenecks in the PET balance. They should probably both be off for ops.

Reply:
I have moved these two lines to config.fcst and set both to .false. for now. Thanks!